1_fetch.yml

```diff
@@ -37,22 +39,28 @@ targets:
   percentiles:
     command: viz_config[[I('percentiles')]]
 
+  # -- view polygon --
+  view_config:
+    command: viz_config[I(c('bbox', 'projection', 'width', 'height'))]
```

Comment:
are we keeping bbox just in case we want to run this for a smaller region in the future? Just wondering if we want to get rid of it or not since this is for the whole US. I like the flexibility we might get by leaving it in, and it might be more work to remove it.